app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    pass

```

Comment:
Consider adding a default error message to the `NotVaccinatedError` class. This can be done by overriding the `__init__` method and providing a default message.